utils/update_checkout/update-checkout-config.json

```diff
@@ -93,7 +93,7 @@
                 "llbuild": "main",
                 "swift-tools-support-core": "main",
                 "swiftpm": "main",
-                "swift-argument-parser": "1.0.3",
+                "swift-argument-parser": "1.1.2",
```

Comment:
Would you mind updating this to 1.1.4, as that version had a few nice improvements on top?

Reply:
Done